libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/dashboardDnD.spec.ts

```diff
@@ -0,0 +1,116 @@
+// (C) 2022 GoodData Corporation
+
+import * as Navigation from "../../tools/navigation";
+import { EditMode } from "../../tools/editMode";
+import { AttributeFilter, FilterBar } from "../../tools/filterBar";
+import { DashboardHeader } from "../../tools/dashboardHeader";
+import { DashboardMenu } from "../../tools/dashboardMenu";
+import { WidgetDropZone } from "../../tools/enum/DropZone";
+import { Widget } from "../../tools/widget";
+import { Dashboard } from "../../tools/dashboards";
+import { WidgetConfiguration } from "../../tools/widgetConfiguration";
+
+Cypress.Cookies.defaults({
+    preserve: ["GDCAuthTT", "GDCAuthSTT", "_csrfToken"],
+});
+
+Cypress.on("uncaught:exception", (error) => {
+    console.error("Uncaught exception cause", error);
+    return false;
+});
+
+Cypress.Cookies.debug(true);
+
+const dashboardHeader = new DashboardHeader();
+const editMode = new EditMode();
+const filterBar = new FilterBar();
+const dashboardMenu = new DashboardMenu();
+const widget = new Widget(0);
+const dashboard = new Dashboard();
+
+describe("Dashboard Drag and Drop", { tags: ["pre-merge_isolated_bear"] }, () => {
+    beforeEach(() => {
+        cy.login();
+    });
+
+    describe("Delete attribute filter", () => {
+        it("Display trash when performing filter drag", () => {
+            Navigation.visit("dashboard/attribute-filtering");
+            editMode.edit();
+
+            widget.waitTableLoaded();
+            new AttributeFilter("Activity Type").dragFilter().hasDeleteDropzone();
+        });
+
+        it("Delete filter when having multiple attribute filters", () => {
+            Navigation.visit("dashboard/multiple-filters");
+            editMode.edit();
+
+            new AttributeFilter("Account").removeFilter();
+            new WidgetConfiguration(0).open().openConfiguration().isFilterItemVisible("Account", false);
+        });
+
+        it("Add deleted attribute filter", () => {
+            Navigation.visit("dashboard/attribute-filtering");
+            editMode.edit();
+
+            new AttributeFilter("Activity Type").removeFilter();
+            filterBar.hasAttributeLength(0);
+
+            cy.wait(1000);
```

Comment:
after remove filter and check filterbar has 0 attribute length, why we still need to wait 1s next?

Reply:
cypress runs too fast, two dnd actions continuously, if not add wait between them, the second actions will be broken, in this case, after dragging filter out then dragging filter in, if these actions happen too fast then the filter cannot be dragged to place holder

Reply:
because it is flaky, I will remove all cy.wait, then keep in track, if it is failed then I will add `wait` back later

Reply:
if there is no wait here, sometime test failed, but not always, remove `wait` here then keep this test in track, if it is failed in 3 times run then I will comeback to it to add `wait`